CK3toEU4/Source/CK3World/Religions/Faith.h

```diff
@@ -41,7 +41,7 @@ class Faith: commonItems::parser
 	std::string customAdjective;
 	std::string description;
 	std::optional<commonItems::Color> color;
-	std::set<std::string> doctrines;
+	std::vector<std::string> doctrines;
```

Comment:
leave set. set is good. you don't want to repeat doctrines and you can poke it with contain().

Reply:
I want them inputted in a specific order. The tenets should be inputted first, since I care more about them then the later doctrinal stuff. (Now that I put a limit on how many modifiers a religion can have, this has become important.)

Reply:
This is perfectly fine, but at least put a header in doctrine_definitions.txt explaining that order matters.
 You can ignore/resolve other issues related to this vector/set.

Reply:
The doctrines are just read top-bottom. They are already pre-organized by what type of doctrine (Tenet vs Other).